### PR TITLE
Remove the wdltool functionality from Cromwell

### PR DESCRIPTION
**Pull request by geoffjentry**

### CODE REVIEW

**Comment by geoffjentry** [comment]
@mcovarr @scottfrazer

**Comment by mcovarr** [comment]
:+1:

**Comment by geoffjentry** [comment]
I need to make an update to changelog

File: src/test/scala/cromwell/MainSpec.scala [hunk 1]
```diff
@@ -35,105 +35,6 @@ class MainSpec extends FlatSpec with Matchers with BeforeAndAfterAll with TimeLi
...
-    testWdl(ThreeStep) { wdlAndInputs =>
-      val wdl = wdlAndInputs.wdl
-      val result = traceAction("inputs", wdl)
-      assert(result.out.contains("\"three_step.cgrep.pattern\""))
-      result.returnCode should be(0)
-    }
-  }
-
-  it should "not return inputs when there is no workflow" in {
-    testWdl(EmptyTask) { wdlAndInputs =>
```
Comment (kshakir) [inline src/test/scala/cromwell/MainSpec.scala]: Remove `EmptyTask` cruft from `SampleWdl`.

File: src/test/scala/cromwell/MainSpec.scala [hunk 2]
```diff
@@ -35,105 +35,6 @@ class MainSpec extends FlatSpec with Matchers with BeforeAndAfterAll with TimeLi
...
-    testWdl(ThreeStep) { wdlAndInputs =>
-      val wdl = wdlAndInputs.wdl
-      val result = traceAction("validate", wdl)
-      result.out should be(empty)
-      result.returnCode should be(0)
-    }
-  }
-
-  it should "not validate invalid wdl" in {
-    testWdl(EmptyInvalid) { wdlAndInputs =>
```
Comment (kshakir) [inline src/test/scala/cromwell/MainSpec.scala]: Remove `EmptyInvalid` cruft from `SampleWdl`.

**Comment by geoffjentry** [comment]
@kshakir cruft removed

**Comment by scottfrazer** [comment]
:+1:

**Pull request merged**